scss/popup.scss

```diff
@@ -33,8 +37,7 @@ header {
 }
 
 main {
-  min-height: $window-height - 2 * $window-padding;
-  min-width: $window-width - 2 * $window-padding;
+  padding: $window-padding;
```

Comment:
If the status is more than one line, the image will shrink. Not sure if it's a bug or a feature